Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/GatewayService.java

```diff
@@ -113,38 +113,37 @@ public static GatewayService getInstance() {
      *
      * @return true if the patient is found else false
      */
-    public boolean discoverPatient(final PatientSearchBean patientQuerySearch) {
+    public boolean discoverPatient(PatientSearchBean patientQuerySearch) {
 
-        // Create the patient bean that needs to be passed to the service layer
-        final gov.hhs.fha.nhinc.patientdiscovery.model.Patient patientBean = new gov.hhs.fha.nhinc.patientdiscovery.model.Patient();
-        // set the orgainization
+        //Create the patient bean that needs to be passed to the service layer
+        gov.hhs.fha.nhinc.patientdiscovery.model.Patient patientBean = new gov.hhs.fha.nhinc.patientdiscovery.model.Patient();
+        //set the orgainization
         patientBean.setOrganization(patientQuerySearch.getOrganization());
-        // set birth date in YYYYMMDD
-        final DateFormat df = new SimpleDateFormat(UTCDateUtil.DATE_ONLY_FORMAT);
+        //set birth date in YYYYMMDD
+        DateFormat df = new SimpleDateFormat(UTCDateUtil.DATE_ONLY_FORMAT);
         patientBean.setBirthDate(df.format(patientQuerySearch.getDateOfBirth()));
-        // set the first name
+        //set the first name
         patientBean.setFirstName(patientQuerySearch.getFirstName());
-        // set last name
+        //set last name
         patientBean.setLastName(patientQuerySearch.getLastName());
-        // set the gender
+        //set the gender
         patientBean.setGender(patientQuerySearch.getGender());
 
         try {
-            // Call the entity/gateway Patient Discovery service
-            final PatientSearchResults patientDiscoveryResults = patientService.queryPatient(patientBean);
-            LOG.debug("Patient Discovery call successful. Total number of patients found: {}",
-                    patientDiscoveryResults.getPatientList().size());
+            //Call the entity/gateway Patient Discovery service
+            PatientSearchResults patientDiscoveryResults = patientService.queryPatient(patientBean);
+            LOG.debug("Patient Discovery call successful. Total number of patients found:" + patientDiscoveryResults.getPatientList().size());
 
-            // Return false if no patient found
+            //Return false if no patient found
             if (patientDiscoveryResults.getPatientList().isEmpty()) {
                 return false;
             }
-            // populate the UI patient object with the results data
+            //populate the UI patient object with the results data
             populatePatientBean(patientDiscoveryResults, patientQuerySearch);
             return true;
-        } catch (final Exception ex) {
```

Comment:
There are several places that "final" was removed -- did the formatter do this, or was this done manually?